translation_fr.md

```diff
@@ -0,0 +1,53 @@
+# Translation Guidelines
+
+The translation of the TC39 web page aims to make the work of TC39 more accessible to people whose
```

Comment:
is this page file meant to be written in English?

Reply:
I think English would be better.